src/dvc_render/vega_templates.py

```diff
@@ -508,16 +501,35 @@ class SmoothLinearTemplate(Template):
                     },
                     "color": {"field": "rev", "type": "nominal"},
                 },
-                "layer": [
+                "transform": [
                     {
-                        "mark": {
-                            "type": "line",
-                            "point": True,
-                            "tooltip": {"content": "data"},
-                        }
-                    }
+                        "loess": Template.anchor("y"),
+                        "on": Template.anchor("x"),
+                        "groupby": ["rev", "filename"],
+                        "bandwidth": {"signal": "smooth"},
+                    },
                 ],
-            }
+            },
+            {
+                "mark": {
+                    "type": "point",
+                    "tooltip": {"content": "data"},
```

Comment:
does it mean that tooltip now is only visible if I hover a point? not on the line itself?

Reply:
I think it was already true. Here's the behavior in `main`:


https://user-images.githubusercontent.com/2308172/215193257-cbf4525a-3bb3-4243-a9cf-d014a3faca59.mov

We could add them on the lines, but I don't think it's worth it. They are finnicky (they still only appear over the x coordinates of the points) and they only give you the loess-transformed y values, which I don't think are particularly interesting.

https://user-images.githubusercontent.com/2308172/215193610-6cb0202c-43d5-4b03-ba7e-fc6e5f5d99e8.mov